### PR TITLE
MGMT-1859: use CapabilityReview via SDK

### PR DESCRIPTION
**Pull request by danielerez**
Use CapabilityReview API from ocm-sdk-go instead of a manually generated request.

### CODE REVIEW

**Comment by danielerez** [comment]
@rollandf @ronniel1 @avishayt @romfreiman

File: pkg/ocm/authorization.go [hunk 1]
```diff
@@ -51,46 +50,39 @@ func (a authorization) AccessReview(ctx context.Context, username, action, resou
...
 	}
-	request.Bytes(jsonData)
 
-	postResp, err := request.SendContext(ctx)
+	postResp, err := capabilityReview.Post().
+		Request(request).
+		SendContext(ctx)
 	if err != nil {
 		return false, err
 	}
```
Comment (ronniel1) [inline pkg/ocm/authorization.go:76]: See @rollandf  review on OCM SDK. He claims that response.status() should be checked

Comment (danielerez) [reply pkg/ocm/authorization.go:76]: I'm not sure it's required, according to the examples it seems unneeded:
https://github.com/openshift-online/ocm-sdk-go/blob/356e35e01902f8e68e6b2353ffd59ed06becc03a/examples/client_credentials_grant.go#L64

We should get an error for 'status >= 400' on sendConext:
https://github.com/openshift-online/ocm-sdk-go/blob/b3f1fc3136d749c108f319f8af5b1647923f4d49/authorizations/v1/capability_review_client.go#L134

Comment (rollandf) [reply pkg/ocm/authorization.go:76]: @danielerez is right no need for it

**Review by rollandf — APPROVED** [review]
:+1:

**Pull request merged**